### ISSUE TITLE
Space Cadet Shift/Ctrl not working with non ANSI Layout

### ISSUE THREAD

**Issue by exen904** [vial-kb/vial-qmk#157]
Hi, if I select another keyboard layout then QWERTY (german in my case) the keyboard layout itself does work fully as expected. Other features, like Space Cadet Shift, Ctrl. etc. dont work as expected. In german layout the `( )` change to `8` and `9`, so `LCtrtl` goes to `)`. Is it possible to fix that? I tried to set a `any` keycode with `LCtl_T` but cant enter multiple keycodes then.

**Comment by lesshonor** [comment]
Non-US ANSI layouts require additional configuration pre-compile. Please refer to the documentation:
https://github.com/vial-kb/vial-qmk/blob/vial/docs/feature_space_cadet.md#configuration
